dev-packages/rollup-utils/bundleHelpers.mjs

```diff
@@ -124,7 +140,7 @@ export function makeBaseBundleConfig(options) {
   const bundleTypeConfigMap = {
     standalone: standAloneBundleConfig,
     addon: addOnBundleConfig,
-    node: nodeBundleConfig,
```

Comment:
Is node not used anywhere else? Nice!

Reply:
Nope, the only thing we bundle on the node side is the aws lambda layer bundle